.riot/requirements/12135c7.txt

```diff
@@ -4,104 +4,115 @@
 #
 #    pip-compile --allow-unsafe --no-annotate .riot/requirements/12135c7.in
 #
-ai21==3.0.1
-ai21-tokenizer==0.12.0
-aiohappyeyeballs==2.4.6
-aiohttp==3.10.11
-aiosignal==1.3.2
+ai21==4.0.3
+aiohappyeyeballs==2.6.1
+aiohttp==3.12.15
+aiohttp-retry==2.9.1
+aiosignal==1.4.0
 annotated-types==0.7.0
-anthropic==0.45.2
-anyio==4.8.0
+anthropic==0.60.0
+anyio==4.9.0
 async-timeout==4.0.3
-attrs==25.1.0
-boto3==1.36.17
-botocore==1.36.17
-certifi==2025.1.31
-charset-normalizer==3.4.1
-cohere==5.13.12
-coverage[toml]==7.6.12
+attrs==25.3.0
+boto3==1.40.1
+botocore==1.40.1
+certifi==2025.7.14
+cffi==1.17.1
+charset-normalizer==3.4.2
+cohere==5.16.1
+coverage[toml]==7.10.1
 dataclasses-json==0.6.7
 distro==1.9.0
-exceptiongroup==1.2.2
-fastavro==1.10.0
-filelock==3.17.0
-frozenlist==1.5.0
-fsspec==2025.2.0
+exceptiongroup==1.3.0
+fastavro==1.12.0
+filelock==3.18.0
+frozenlist==1.7.0
+fsspec==2025.7.0
 greenlet==3.0.3
-h11==0.14.0
-httpcore==1.0.7
-httpx==0.27.2
+h11==0.16.0
+hf-xet==1.1.5
+httpcore==1.0.9
+httpx==0.28.1
```

Comment:
@quinna-h I think there's an upstream issue with httpx and client proxies. Can you try pinning the httpx dependency to `0.27.2`?